From c8772e690ef1b21ae4f4d366971a6a0536a4ce34 Mon Sep 17 00:00:00 2001
From: xqq <xqq@0ginr.com>
Date: Sun, 11 Oct 2015 16:44:27 +0800
Subject: [PATCH] vcxproj: Static link runtime library on Release build

---
 libwtfdanmaku.vcxproj | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/libwtfdanmaku.vcxproj b/libwtfdanmaku.vcxproj
index 9a92647..7156189 100644
--- a/libwtfdanmaku.vcxproj
+++ b/libwtfdanmaku.vcxproj
@@ -123,6 +123,7 @@
       <PreprocessorDefinitions>WIN32;NDEBUG;_WINDOWS;_USRDLL;LIBWTFDANMAKU_EXPORTS;$(AdditionalDefs);%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <SDLCheck>true</SDLCheck>
       <AdditionalIncludeDirectories>$(ProjectDir)3rdparty;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
     </ClCompile>
     <Link>
       <SubSystem>Windows</SubSystem>
@@ -141,6 +142,7 @@
       <PreprocessorDefinitions>WIN32;NDEBUG;_WINDOWS;_USRDLL;LIBWTFDANMAKU_EXPORTS;$(AdditionalDefs);%(PreprocessorDefinitions)</PreprocessorDefinitions>
       <SDLCheck>true</SDLCheck>
       <AdditionalIncludeDirectories>$(ProjectDir)3rdparty;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <RuntimeLibrary>MultiThreaded</RuntimeLibrary>
     </ClCompile>
     <Link>
       <SubSystem>Windows</SubSystem>
-- 
2.5.0.windows.1

